CHANGELOG.md

```diff
@@ -12,8 +12,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Enable double encoding for all RQL operations with `rqlBuilder.doubleEncodeValues = true`
   - Enable double encoding for a single RQL operation `rqlBuilder({ doubleEncodeValues: true }).eq(name, '< value >').build()`
   - When enabling double encoded values ensure that instances of encoding values for the rql builder such as `encodeURIComponent()` are removed
-- A skip count operator to the rql builder `rqlBuilder().eq(name, '< value >').skipCount().build()` 
+  - Double Encoding of RQL values is currently only supported in `OAuth2` clients and will come to `OAuth1` clients at a later date.
+
+
+- A skip count operator to the rql builder `rqlBuilder().eq(name, '< value >').skipCount().build()`
```

Comment:
(minor) I would leave the `.eq(name, '< value >')` out of it, but no nothing strong against it